runtime/bootstrap/index.ts

```diff
@@ -242,6 +243,7 @@ export async function injectExecutionContext({
   timezone: string;
   invocationToken?: string;
   sync?: Sync;
+  executionId?: string;
```

Comment:
Can you do this to catch this for next time?

```ts
export async function injectExecutionContext({
  context,
  fetcher,
  temporaryBlobStorage,
  logger,
  endpoint,
  invocationLocation,
  timezone,
  invocationToken,
  sync,
  executionId,
  ...rest
}: ExecutionContext & {context: Context; logger: Logger}): Promise<void> {
  ensureNever<keyof typeof rest>();
  ...
```

Reply:
I don't think that works? 
![Screenshot 2024-09-09 at 3 21 43 PM](https://github.com/user-attachments/assets/9c4bee6d-ed47-4805-b4d2-e9e8eabeb292)

Maybe you mean something like this?
![Uploading Screenshot 2024-09-09 at 3.25.02 PM.png…]()

Reply:
oops didn't wait for upload
![Screenshot 2024-09-09 at 3 25 30 PM](https://github.com/user-attachments/assets/4b5df734-43d7-46d9-bd6c-12382b209719)